test/hotspot/jtreg/compiler/arguments/TestTraceICs.java

```diff
@@ -25,8 +25,7 @@
  * @test
  * @bug 8217447
  * @summary Test running TraceICs enabled.
- * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -XX:+TraceICs
- *                   compiler.arguments.TestTraceICs
+ * @run main/othervm compiler.arguments.TestTraceICs
```

Comment:
I think you need to add `-Xlog:inlinecache=trace` flag to keep previous test's behavior and check your new code.